dashboard/db/migrate/20210908195236_update_backpack_storage_apps_value.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateBackpackStorageAppsValue < ActiveRecord::Migration[5.2]
+  def change
+    Backpack.all.each do |backpack|
+      storage_app_row = PEGASUS_DB[:storage_apps].where(id: backpack.storage_app_id).first
+      if storage_app_row[:value] == "".to_json
```

Comment:
Is it possible to have other values?  If so, do we need to append `hidden: true` to them?

Reply:
In theory it shouldn't be since we pass an `''` when creating a new encrypted_id for each Backpack, but we could check just to be safe

Reply:
It doesn't look like there are any values other than "" currently in production.  I'm ok with keeping this simple if we don't think there are other cases.

Reply:
I did a quick query, currently for storage apps with project type backpack the only value is `""`

Reply:
Sorry, just to clarify, I think it would be good to still check for `""` so we don't accidentally delete data that we needed.  I don't think we need to add code to handle merging `hidden: true` into existing data (because all known rows don't have data).

Reply:
Right, sorry I responded to the initial comment without reloading the PR so I didn't see that ya'll had already done the investigation.